corehq/apps/sms/tests/__init__.py

```diff
@@ -392,4 +395,14 @@ def test_backend(self):
         self.backend3.delete_invoke_doc()
         self.assertFalse(self.backend3.invoke_doc_exists())
 
+    def test_sms_registration(self):
+        incoming("+9991234567", "JOIN {} WORKER tester".format(self.domain), "BACKEND1")
```

Comment:
Why would this be "TEST"? In the test setup we create a TestBackend named "BACKEND1" https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/sms/tests/__init__.py#L60